Lib/Rabbit4000/BIOSLIB/CPUPARAM.LIB

```diff
@@ -122,8 +122,6 @@ _enableClockModulation::
 	bool	hl
 #if ENABLE_SPREADER == 0	// spreader disabled
 	xor	a
-	ld		(MTCRShadow), a
-	ioi	ld (MTCR), a		// normal /OE0 and /OE1 timing

```

Comment:
Even with the spreader disabled, we should keep the "early /OE" setting on products that have enabled it.